File: lib/js_test_server/server/resources.rb
dir = File.dirname(__FILE__)

module JsTestServer::Server::Resources
end

require "#{dir}/resources/resource"
require "#{dir}/resources/file"
require "#{dir}/resources/spec_file"
require "#{dir}/resources/framework_file"
require "#{dir}/resources/web_root"
require "#{dir}/resources/not_found"
require "#{dir}/resources/remote_control"

require "#{dir}/resources/implementations_deprecation"

File: lib/js_test_server/server/resources.js
var sys = require('sys'),
    express = require('express');

JsTestServer.Server.Resources = {}

require("./resources/file_dispatch")

require("./resources/web_root").bind()
require("./resources/file").bind()
require("./resources/framework_file").bind()
require("./resources/spec_file").bind()
require("./resources/not_found").bind()
